doc/pages/press.json

```diff
@@ -1,5 +1,10 @@
 [
   {
+    "link_name":"Making it through tough times together",
+    "link":"http://sunthisweek.com/2017/09/22/making-it-through-tough-times-together/",
+    "author":"By: Laura Adelmann"
```

Comment:
Thanks for adding this :) 
Could you update "By: Laura Adelmann" to "By: Laura Adelmann, SunThisweek"? :D